openedx/core/djangoapps/content/course_overviews/migrations/0008_drop_course_tabs_fk_constraint.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Foreign keys aren't enforced by default on our version of SQLite3, and
+        # trying to delete them will throw an error. See:
+        #   http://south.aeracode.org/ticket/775
+        if db.backend_name != 'sqlite3':
+            db.delete_foreign_key('course_overviews_courseoverviewtab', 'course_overview_id')
```

Comment:
Django will still treat it like a foreign key (the relations will work). And the tabs will get deleted when you call `delete()` on the `CourseOverview` they're linked to (just verified that locally). This just kills the database layer constraint.